nifi-docker/1.1.1/DockerRun.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+DOCKER_IMAGE="$(cat DockerImage.txt)"
+docker run -it -d -p 8080:8080 -p 8181:8181 $DOCKER_IMAGE
```

Comment:
Do we have to execute following command every time we want to start nifi docker instance?
 docker run -it -d -p 8080:8080 -p 8181:8181 $DOCKER_IMAGE 

or once docker container is created, we can do docker stop and docker start.
i am trying that with my docker container. It works well with docker run. but it does work well with docker stop and docker start.

Can you try that?